applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -275,6 +372,14 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 		return;
 	}
 
+	headsets[channel_index].pref_qos = *pref;
+
+	find_heaset_pres_delay(false, channel_index, &pd, &pref_pd);
```

Comment:
Sending a false requires the user to check the function definiton to see what is going on. See my earlier comment, or we may need to create an enum with FIND_MAX and FIND_MIN

Reply:
See my earlier comment, would prefer to create a type to make it clear.

Reply:
No longer relevant.